lib/packages.ts

```diff
@@ -28,53 +26,19 @@ export interface PackageInfo {
   build: string;
   tar: string;
   private: boolean;
+  experimental: boolean;
```

Comment:
does this change mean we can update release.md to stop using --force?

Reply:
Yes, done.